compair/static/modules/assignment/assignment-view-partial.html

```diff
@@ -247,13 +247,29 @@ <h2>All answers
                         <strong>said</strong> on {{answer.created | amDateFormat: 'MMM D @ h:mm a'}}:
                         <div class="manager-actions pull-right" ng-if="canManageAssignment">
                             <em>{{instructors[answer.user_id]}} Response</em>
+                            <em ng-if="answer.comparable">(Comparable)</em>
```

Comment:
look for `.order_by(desc('instructor_role'), desc('ta_role'))` in `AnswerRootAPI`

Reply:
As discussed, will also change the UI logic to sort instructor answers and student answers together when ordering by answer rank.

Will change the `ng-if` clause to include checking of `answerFilters.orderBy`.
```
                <!-- Instructor Answers Section -->
                <div class="each-answer clearfix" ng-repeat="answer in answers.objects | filter:adminFilter() as adminResults" ng-if="answerFilters.orderBy!='score' && instructors[answer.user_id] && (see_answers || canManageAssignment)">
```

Will also modify the `adminFilter()` function to keep comparable answers from instructors.

Reply:
one thing I missed before is I'm not sure if we want to label the answer as "Comparable" or not. Its something I'd want to check with @lenglund about but since since that will have to wait until next week, I think it would be safer to not include it for now (and add it back later). Maybe make an issue `1.1.1` issue on it